addons/website_sale/views/templates.xml

```diff
@@ -1248,7 +1248,7 @@
         <xpath expr="//a[@id='add_to_cart']" position="after">
             <a role="button" t-attf-class="btn btn-outline-primary o_we_buy_now ms-1 #{'btn-lg' if ctaSizeBig else ''}" href="#">
                 <i class="fa fa-bolt me-2"/>
-                <span style="font-weight:bold">BUY NOW</span>
```

Comment:
Thanks to @mano-odoo and its redesign of the product checkout, this will no longer be necessary: 
https://github.com/odoo/odoo/pull/113537/files#diff-635040c730ccbb8e4f6f2bf6472cf48a0cc9b8a3d488289dabdfce8096811915L975
(this PR is still in draft form and is currently being splitted)

So I guess we could remove it here ?

Reply:
> So I guess we could remove it here ?

I think it should stay since it has an huge visual impact.

Both PR aim for the same visual result; assuming that this one will be merged first,  @mano-odoo PR will eventually adapt accordingly.

Reply:
> PR will eventually adapt accordingly.

Ok then I'll merge this change and you can adapt and merge the other one with the eCommerce team :+1: